### PR TITLE
Feature/cypress donate milk 2 dw

### PR DESCRIPTION
**Pull request by devonwigle**
Is this a fix or a feature?
- [ ] Bug fix
- [x] Feature
- [ ] Refactor
 
Describe the feature - fix - refactor:

Cypress testing for donating milk flow
Add test ids 

How was this tested?
- [ ] open index.html
- [x] localhost
- [ ] terminal

Where should the reviewer start - test:

npm start
in separate terminal, npm run cypress
view testing files

Additional comments:

### CODE REVIEW

**Comment by janitastic** [comment]
This has been updated in a different PR using your changes on this one.

File: cypress/integration/donate_milk_spec.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+describe('donate milk form user flow', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/donate')
+  });
+
+  it('should get a sorry message when yes to smoking is selected', () => {
```
Comment (cagallo) [inline cypress/integration/donate_milk_spec.js:6]: Smart to add this back in so we can test the inputs with actual user data.

**Comment by janitastic** [comment]
I removed the unnecessary testids, and added back in the "on page load" tests.

**Comment by cagallo** [comment]
Great job on this test!

**Pull request merged**